### PR TITLE
MAINT: warns for new multichannel default in denoise_{bilateral, nl_means}

### LINKED ISSUE #2465: Change API denoise_bilateral, multichannel=False by default.
## Description

See https://github.com/scikit-image/scikit-image/pull/2461#issuecomment-273802520

in skimage/restoration/_denoise.py, deprecate mutlichannel=True to False for denoise_bilateral to be consistent with the rest of the functions. Arguments are listed in the link at the top of this message.

In doc/examples/filters/plot_denoise.py, the function call must be adapted as well.

### PR DESCRIPTION
**Pull request by stsievert**
Fixes #2465

## Description
In https://github.com/scikit-image/scikit-image/pull/2461#issuecomment-273802520 we realized that `denoise_wavelet` should default to `multichannel=False`. We chose this primarily because `multichannel=False` works well for 2D images, is explicit and doesn't prioritize multichannel images (e.g., RGB images) over 2D images. 

From this we decided that `denoise_bilateral` should default to `multichannel=False` to provide a consistent interface with the multichannel found in `denoise_{wavelet, tv_chambolle}`. `denoise_nl_means` also defaults to `multichannel=True` and though not discussed this is changed as well.

## Checklist
- [x] denoise_bilateral
    - [x] multichannel warning
    - [x] example changed
- [x] denoise_nl_means
    - [x] multichannel warning
    - [x] examples changed

## References
This closes #2465.

### CODE REVIEW

**Comment by sciunto** [comment]
@stsievert You need to follow the procedure that we started to write here: #2422 

Thx!

File: skimage/restoration/_denoise.py [hunk 1]
```diff
@@ -75,7 +75,10 @@ def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
     >>> noisy = np.clip(noisy, 0, 1)
     >>> denoised = denoise_bilateral(noisy, sigma_color=0.05, sigma_spatial=15)
     """
-    warn('denoise_bilateral will default to multichannel=False')
+    if multichannel is None:
+        warn('denoise_bilateral will default to multichannel=False')
```
Comment (sciunto) [inline skimage/restoration/_denoise.py]: Please, specify the version here (0.15).

**Comment by sciunto** [comment]
At the end, if you can squash the commit, would be awesome ;)

**Comment by sciunto** [comment]
You also need to update the test suite and specify the option everywhere.

**Comment by stsievert** [comment]
Rebased, and the tests passed before the rebase.

**Comment by sciunto** [comment]
@scikit-image/core We need a second review here. It's easy :)

**Comment by jni** [comment]
Thanks @stsievert! and @sciunto, you were right, that was easy. =D

**Pull request merged**